docs/apache-airflow/installation/index.rst

```diff
@@ -316,3 +327,66 @@ Follow the  `Ecosystem <https://airflow.apache.org/ecosystem/>`__ page to find a
 **Where to ask for help**
 
 * Depends on what the 3rd-party provides. Look at the documentation of the 3rd-party deployment you use.
+
+
+Notes about minimum requirements
+''''''''''''''''''''''''''''''''
+
+There are often questions about minimum requirements for Airflow for Production systems, but it is
+not possible to give a simple answer to that question.
+
+The requirements that Airflow might need depend on many factors, including (but not limited to):
+  * The deployment your Airflow is installed with (see above ways of installing Airflow)
+  * The requirements of the Deployment environment (for example Kubernetes, Docker, Helm, etc.) that
+    are completely independent from Airflow (for example DNS resources, sharing the nodes/resources
+    with more (or less) pods and containers that are needed that might depend on particular choice of
+    the technology/cloud/integration of monitoring etc. etc.
+  * Technical details of database, hardware, network, etc. that your deployment is running on
+  * The complexity of the code you add to your DAGS, configuration, plugins, settings etc. (note, that
+    Airflow runs the code that DAG author and Deployment Manager provide)
+  * The number and choice of providers you install and use (Airflow has more than 80 providers that can
+    be installed by choice of the Deployment Manager. Also they are use more or less resources when
+    they are used - in various scenarios - parsing, webserver, task execution
```

Comment:
The latter sentence needs a little re-wording I think!

Reply:
Yeah. Not finished/complete. Shortened and merged with the previous one.